personalize.module

```diff
@@ -669,6 +671,17 @@ function personalize_library() {
       array('system', 'jquery.once'),
     ),
   );
+  $libraries['admin.campaign.list'] = array(
+    'title' => 'Personalization list',
+    'website' => '',
+    'version' => VERSION,
+    'js' => array(
+      'jQuery(document).ready(function(){jQuery(".content").tabs();});' => $campaign_list_options,
```

Comment:
Granted... this only runs when the library is included and that is only on one page for the time being, I think you want a slightly more specific selector.  Typically modules prefix things like that with the module machine name to avoid naming collisions.